src/iso3166/__init__.py

```diff
@@ -490,7 +519,10 @@ def get(
             r = _by_numeric.get(k, default)
         else:
             k = key.upper()
-            if len(k) == 2:
+            if len(k) >= 2 and k[0] == "+": 
+                print("cock")
```

Comment:
?

Reply:
Ops, I forgot to remove THIS debugging print statement that I used to fix a bug. Now the code should be ok, sorry for the inconvinient.